admin_guide/high_availability.adoc

```diff
@@ -0,0 +1,132 @@
+
+Running with the `--help` option will show you the supported options and their description. A small subset of these options pertinent to this document is described in the next section.
+
+****
+`$ oscadm ipfailover --help`
+****
+
+=== ipfailover command syntax
+
+        oscadm ipfailover [<name>] <options>
+        where:
+            <name> = Name of the IP failover configuration.
+                     Default: generated name (e.g.  ipfailover-1)
+            <options> = One or more options - see subset below.
+
+=== ipfailover command options (subset)
+The list of command options described here are a subset that are relevant to this document.
+
+            <options> = One or more of:
+                --create
+                --credentials=<credentials>
+                -l,--selector=<selector>
+                --virtual-ips=<ip-range>
+                -i|--interface=<interface>
+                -w|--watch-port=<port>
+
+            <credentials> = <string> - Path to .kubeconfig file containing credentials to use to contact the master.
+            <selector> = <string> - The node selector to use for running the HA sidecar pods.
+            <ip-range> = string - One or more comma separated IP address or ranges.
+                                  Example: 10.2.3.42,10.2.3.80-84,10.2.3.21
+            <interface> = <string> - The interface to use.
+                                     Default: Default interface on node or eth0
+            <port> = <number> - Port to watch for resource availability.
+                                Default: 80.
+            <string> = a string of characters.
+            <number> = a number ([0-9]*).
+
+
+== Setting up Routers with IP Failover
+The following steps describe how to setup a highly available router environment with IP failover in a 3-step operation:
+
+=== Step 1: Label the infrastructures nodes for the service (router)
+Strictly speaking, this step can be optional as you can run the router instances on any of the nodes in your Kubernetes cluster and use Virtual IP addresses (VIPs) that can "float" within those nodes.
+
+However that said, it is recommended you provision certain infrastructure nodes to run the routers and have VIPs that can "float" amongst these nodes. In a complex and possibly bigger cluster, you probably may be already doing something similar so that nodes may be filtered on constraints or requirements specified (e.g. nodes with SSD drives or higher cpu/memory/disk requirements, etc).
+
+In our example, let us define this label or constraint as router instances servicing traffic in the US west geography "ha-router=geo-us-west".
+
+****
+`$ openshift kube label nodes openshift-minion-{5,6,7,8,9} "ha-router=geo-us-west"`
```

Comment:
Reminder for docs writer following up: Update this to `osc label` after https://github.com/openshift/openshift-docs/pull/332 merges.